src/adapters/simple_interest_loan_adapter.ts

```diff
@@ -68,7 +68,7 @@ export const SimpleInterestAdapterErrors = {
                          WEEKS, MONTHS, or YEARS.`,
     INVALID_TERM_LENGTH: () =>
         singleLineString`Term length value cannot be negative or greater
-                         than 2^120 - 1`,
+                         than ${parseInt(MAX_TERM_LENGTH_VALUE_HEX)}`,
```

Comment:
might want to specify this is seconds?

Reply:
It's not seconds -- it could be in any un it.